src/features/create-quiz/create-quiz-second-card/components/answer-item.tsx

```diff
@@ -27,6 +29,9 @@ export const AnswerItem: React.FC<AnswerItemProps> = ({
           style={{ width: "312px" }}
           className="create-quiz__input"
           placeholder="Enter your answer"
+          onChange={(event) =>
```

Comment:
The function `handleChange` doesn't seem to take in the third parameter, so I think "answerName" could be removed.